lib/ui/compositing.dart

```diff
@@ -191,6 +191,14 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// stack.
   void pop() native 'SceneBuilder_pop';
 
+  /// Add a retained engine layer subtree from previous frames.
+  ///
+  /// All the engine layers that are in the subtree of the retained layer will
+  /// be automatically appended to the current engine layer tree. Therefore,
+  /// once this is called, there's no need to call [addToScene] for its children
```

Comment:
Mentioning `addToScene` here is a bit of a non-sequitur, since it's not part of this API. I think it's fine to mention it, but we'd have to add a lot more context, e.g. referring to "when implementing a subclass of the [Layer] concept defined in the rendering layer of Flutter's framework...".

Reply:
Done.